Xamarin.Forms.Core/Color.cs

```diff
@@ -400,7 +400,27 @@ public static Color FromHsla(double h, double s, double l, double a = 1d)
 		{
 			return new Color(h, s, l, a, Mode.Hsl);
 		}
+
+		int CalculateYiqLuma()
+		{
+			var r = _r * 255;
+			var g = _g * 255;
+			var b = _b * 255;
+			return Convert.ToInt32(Math.Round((299 * r + 587 * g + 114 * b) / 1000f));
+		}
+
+		public bool IsDark
+		{
+			get => CalculateYiqLuma() < 128;
+		}
+
+		public bool IsWhite
+		{
+			get => CalculateYiqLuma() > 128;
```

Comment:
what if value == 128?

Reply:
it will be black

Reply:
but IsDark will return false.
I think it should be
```csharp
public bool IsDark 




		{ 
			get => CalculateYiqLuma() <= 128; 
		}
```

Reply:
Yes you are right

Reply:
The formula in your link considers >= 128 to be light.

Reply:
@jcmanke I had to change this. because we show the property of our color, and not which color we need to use.
In the example, the function returns the color of the text.

Reply:
@jcmanke 
in the test:
``` cs
Assert.IsTrue(Color.FromHex("#FFCC00").IsLight); 
```
This means that our color is light and we need to use dark text.

Reply:
@jcmanke  Thank you for your attention and that you noticed an inverted condition.